src/Symfony/Component/PropertyInfo/Extractor/ReflectionExtractor.php

```diff
@@ -207,6 +207,29 @@ public function getTypesFromConstructor(string $class, string $property): ?array
 
     public function getType(string $class, string $property, array $context = []): ?Type
     {
+        // BC layer, remove context key in 8.0
```

Comment:
Could you update the `CHANGELOG.md` of `PropertyInfo` and `UPGRADE-7.3.md` as well?